clio-linkchecking/src/main/java/eu/europeana/clio/linkchecking/dao/MongoCoreDao.java

```diff
@@ -137,7 +137,7 @@ public Dataset getPublishedDatasetById(String datasetId) throws ClioException {
   public Instant getLastUpdateTime() {
     final ResultList<WorkflowExecution> executions = new WorkflowExecutionDao(datastoreProvider)
             .getAllWorkflowExecutions(null, EnumSet.of(WorkflowStatus.FINISHED),
-                    DaoFieldNames.FINISHED_DATE, false, 0, false);
+                    DaoFieldNames.FINISHED_DATE, false, 0,null, false);
```

Comment:
I think the `pageCount` parameter should be '1' instead of `null`, because we want only one page. We just need the very first result, so there is no point getting more than one page, right?

Reply:
Done